installer/scripts/tomlparser-prom-customconfig.rb

```diff
@@ -40,7 +41,7 @@ def parseConfigMap
       return nil
     end
   rescue => errorStr
-    puts "config::error::Exception while parsing toml config file for prometheus config: #{errorStr}, using defaults"
+    ConfigParseErrorLogger.logError("Exception while parsing toml config file for prometheus config: #{errorStr}, using defaults")
```

Comment:
Please make these messages customer friendly. like : 'Error when parsing configmap. Please check the configMap for errors.'